br/pkg/lightning/restore/check_info.go

```diff
@@ -48,6 +48,7 @@ import (
 	"github.com/pingcap/tidb/table"
 	"github.com/pingcap/tidb/table/tables"
 	"github.com/pingcap/tidb/types"
+	tidbmath "github.com/pingcap/tidb/util/mathutil"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 	"modernc.org/mathutil"
```

Comment:
I think you can remove the dependency of modernc.org/mathutil and replace it with github.com/pingcap/tidb/util/mathutil instead since only some Max/Min functions was used from it anyway.

I could only find it to be used in these files:
br/pkg/lightning/checkpoints/checkpoints.go
br/pkg/lightning/restore/check_info.go
br/pkg/lightning/restore/restore.go